### ISSUE TITLE
Support datasets from the BioImage archive

### ISSUE THREAD

**Issue by IgnacioHeredia** [J535D165/datahugger#78]
Hi @J535D165 ,

I'm trying to download a dataset from the [Bioimage archive](https://www.ebi.ac.uk/biostudies/BioImages/studies), for example [this one](https://www.ebi.ac.uk/biostudies/BioImages/studies/S-BIAD1232). But it raises a non-supported error:

```console
❯ datahugger 10.6019/S-BIAD1232 test-bioimage                                           
Error: Data protocol for 10.6019/S-BIAD1232 not found.

Do you think this is a data repository that needs to be supported?
Please request support in the issue tracker:

	https://github.com/J535D165/datahugger/issues/new/choose
```

However, it looks that the Bioimage Archive is actually [integrated in the Dataverse](https://www.re3data.org/repository/r3d100013949), which is in turn [supported by Datahugger](https://j535d165.github.io/datahugger/repositories/).
So my questions really is: _how can I download a dataset from a Dataverse-supported repository?_

Thanks. :)

**Comment by micafer** [comment]
Hi @IgnacioHeredia,
I think that this repository is not part of the Dataverse installations:
https://dataverse.org/installations

**Comment by IgnacioHeredia** [comment]
Hi @micafer ,

I'm not familiar with how Dataverse works. Reading the [Datahugger docs](https://j535d165.github.io/datahugger/repositories/#dataverse-repositories) I see:
> See https://dataverse.org/institutions and [DataVerse on Re3data.org](https://www.re3data.org/search?query=&software%5B%5D=DataVerse) for an overview of DataVerse repositories.

So as the repository [exists en re3data](https://www.re3data.org/repository/r3d100013949) I assumed it could be used as is with Datahugger. Is this not the case?

**Comment by J535D165** [comment]
It looks like they aren't using dataverse. 

I can see a custom REST API for BioImage (see JSON button). I, or anyone else, can explore that. Rest of it looks straightforward to support.

**Comment by IgnacioHeredia** [comment]
Hi @J535D165 ,
it would be great if the Bioimage archive could be supported! 🚀 
BTW thanks for the tool. :)

**Comment by ctr26** [comment]
Hi all,

Just a heads up, the BIA API won't be in production until mid august. So we'll have to revive this issue then

Apologies,

Craig

**Comment by IgnacioHeredia** [comment]
Hi @ctr26 , any updates on the status of the BIA API?

**Comment by unkcpz** [comment]
hi @IgnacioHeredia, in new project, we rewrite datahugger in rust with boosting the performance and easy extension to support new data repository. Can you reopen the issue at https://github.com/EOSC-Data-Commons/datahugger-ng? thanks!

**Comment by IgnacioHeredia** [comment]
Hi @unkcpz , I'm not sure if BIA has yet a documented API so no need to reopen this issue I'm afraid. I'll keep an eye on your fork, though I think it would be good practice to cite this original repository in your [README](https://raw.githubusercontent.com/EOSC-Data-Commons/datahugger-ng/a29c0b6077ab4ea67f172712125b3d71d591128f/README.md) (haven't found any mention).

**Comment by unkcpz** [comment]
> h I think it would be good practice to cite this original repository in your [README](https://raw.githubusercontent.com/EOSC-Data-Commons/datahugger-ng/a29c0b6077ab4ea67f172712125b3d71d591128f/README.md) (haven't found any mention).

true! thanks for the head up. It was in the "about" description of the repo, I changed it and forget to add it to readme. My bad. My apologize to @J535D165. I'll add it back in read me now. 

UPDATE: here is the ack ;) https://github.com/EOSC-Data-Commons/datahugger-ng?tab=readme-ov-file#ack
